volumina/croppingMarkers.py

```diff
@@ -65,21 +133,41 @@ def set_crop_extents(self, crop_extents):
         self._crop_extents = map(list, crop_extents) # Ensure lists, not tuples
         self.changed.emit( self )
 
+    def set_crop_times(self, crop_times):
+        assert len(crop_times) == 2
+        self._crop_times = [crop_times[0],crop_times[1]]
+        self.changed.emit( self )
+
+    def cropZero(self):
+        if self._crop_extents == None:
+            return True
+
+        flag = True
+        for c in self._crop_extents:
+            flag = (flag and c[0]==0 and c[1]==0)
+
+        return flag
+
 class CroppingMarkers( QGraphicsItem ):
     PEN_THICKNESS = 1
 
     def boundingRect(self):
-        # Return an empty rect to indicate 'no content'
-        # This 'item' is merely a parent node for child items
-        return QRectF()
+        width, height = self.dataShape
+        return QRectF(0.0, 0.0, width, height)
+
```

Comment:
I think this isn't right.  Since this item is marked with the `ItemHasNoContents` flag, you should leave this implementation of `boundingRect()` unchanged (i.e. return an empty `QRect`).  And below, you should delete the implementation of `paint()` entirely, instead of reimplementing it as `def paint(...): pass`.